nl.json

```diff
@@ -139,54 +148,126 @@
                 "form-username" : "Gebruikersnaam",
-                "form-criteria" : "Kies alstublieft een gebruikersnaam tusseen 2 en 32 letters, die kan bestaan uit alfanumerieke letters, laag liggende streepjes en koppeltekens.",
-                "button-text" : "Stel jouw gebruikersnaam vast"
+                "form-criteria" : "Kies een gebruikersnaam met 2 tot 32 tekens, het mag letters, cijfers en de tekens - en _ bevatten",
+                "button-text" : "Gebruikersnaam opslaan"
             },
             "locked" : {
                 "title" : "Kan niet inloggen",
-                "line1" : "Je kon niet worden ingelogd bij fnbr.co omdat [je account gesloten is](class=text-danger)."
+                "line1" : "Je kan niet inloggen op fnbr.co omdat [je account geblokkeerd is](class=text-danger)."
             },
             "migrated" : {
                 "title" : "Account gemigreerd",
-                "line1" : "Je bent inglogd met Twitter als",
-                "line2" : "Maar dat account is gemigrerd naar een Discord login."
+                "line1" : "Je bent ingelogd met Twitter als",
+                "line2" : "Maar dat account is gemigreerd om Discord te gebruiken om te kunnen inloggen."
             },
             "notFound" : {
                 "title" : "Geen account gevonden",
-                "line1" : "Je bent inglogd met Twitter als",
-                "line2" : "Echter, heeft dat account heeft nog nooit eerder ingelogd, we ondersteunen Twitter logins momenteel alleen maar voor migraties.",
-                "line3" : "Als alternatief, kun je [inloggen met Discord](/oauth/discord)."
+                "line1" : "Je bent ingelogd met Twitter als",
+                "line2" : "Maar dat account heeft nog nooit eerder ingelogd op fnbr.co en we ondersteunen Twitter nu alleen nog voor account migratie.",
+                "line3" : "Je kan in plaats daarvan [inloggen met discord](/oauth/discord)."
             }
         },
         "errors" : {
             "404" : {
                 "title" : "Fout",
-                "line1" : "De gevraagde bron bestaat niet of u heeft geen toestemming om deze te bekijken."
+                "line1" : "Je bron die je hebt opgevraagd bestaat niet, of je hebt geen toestemming om het te kunnen bekijken."
             },
             "500" : {
-                "title" : "Uh oh!",
-                "line1" : "Er is iets misgegaan tijdens het verwerken. Probeer het later opnieuw!",
-                "line2" : "Je kan [hier](/oauth) opnieuw proberen in te loggen."
+                "title" : "Oh nee!",
+                "line1" : "Er is iets fout gegaan tijden het verwerken, probeer het later opnieuw!",
+                "line2" : "Je kan proberen om opnieuw in te loggen, klik dan [hier](/oauth)."
             }
         }
     }
-}
```

Comment:
Why did you remove that? It was correct before. Your change breaks the whole file, so it can't be used as a translation.